strawberry/experimental/pydantic/__init__.py

```diff
@@ -1,6 +1,14 @@
 from .error_type import error_type
 from .exceptions import UnregisteredTypeException
 from .object_type import input, interface, type
+from .utils import auto
```

Comment:
We probably want to promote this up to a more accessible package eventually, but I didn't want to leave the `experimental` namespace without input from a maintainer.

Reply:
did you have any id where this could live? :D